### PR TITLE
Added convenience functions for row handling, height and improve performance for rendering

### PR DESCRIPTION
**Pull request by Vinschni**
I added a function to delete all rows, get height, width in a table as my application requires loads different data sets.
Furthermore i added a constraint to only display visible content, which greatly improves renderperformance with row data (e.g. 10 kb or more)

### CODE REVIEW

File: table.go [hunk 1]
```diff
@@ -3,13 +3,14 @@ package stickers
...
 	"fmt"
-	"github.com/charmbracelet/lipgloss"
 	"log"
 	"math"
 	"reflect"
 	"strconv"
 	"strings"
 	"unicode/utf8"
+
+	"github.com/charmbracelet/lipgloss"
```
Comment (76creates) [inline table.go]: please avoid these kind of IDE formatting changes

**Comment by 76creates** [comment]
thanks for contributing! ⚡

File: table.go (hunk 1 above)
Comment (Vinschni) [reply table.go]: Reverted in [b26919d](https://github.com/76creates/stickers/pull/5/commits/b26919dcf53b9c16472021aa1ca2679c71998240)

**Comment by Vinschni** [comment]
Hi @76creates i added further things, I'm eager for some feedback especially on [209072a](https://github.com/76creates/stickers/pull/5/commits/209072a76407f6a38f3f9e51cb4c076f4f995fb8)

**Comment by 76creates** [comment]
there are still pending changes

**Comment by 76creates** [comment]
_stale, closing_

**Pull request closed without merging**